source/elements/oneDAL/source/data_management/tables.rst

```diff
@@ -60,7 +62,7 @@ Each table object in |dal_short_name| follows these requirements:
 1. Table objects in |dal_short_name| are :term:`immutable <Immutability>` (it is not possible
```

Comment:
Use term shall
Table objects shall be immutable 
Builders shall be used to construct table obj
etc

I also see the v0.7 comments should be applied as well 
etc

Reply:
please don't use shell, use should

Reply:
The origin of 'shall' is in the approach defined for formulation of the requirements coming from FuSa space/respective trainings on the reqs.

Reply:
ok